src/components/containers/notification/Notification.js

```diff
@@ -146,6 +146,13 @@ function renderCallsToAction(callsToAction) {
       {callsToAction.map((callToAction, index) => {
         const buttonStyleType = index === 0 ? 'primary' : 'default';
 
+        if (React.isValidElement(callToAction)) {
```

Comment:
What worries me about this is that there's no guarantee that this will be a Button component, which kind of defeats the purpose of having the calls to action be part of the Notification component, no?